src/main/java/org/cactoos/func/FuncOf.java

```diff
@@ -95,7 +95,7 @@ public FuncOf(final Proc<X> proc, final Y result) {
      * Ctor.
      * @param fnc Func
      */
-    public FuncOf(final Func<X, Y> fnc) {
+    private FuncOf(final Func<X, Y> fnc) {
```

Comment:
@yegor256 I'm sorry, but why make this constructor as private? You are not going to against Cactoos principles, are you?